src/pages/home/report/ThreadDivider.tsx

```diff
@@ -48,7 +47,6 @@ function ThreadDivider({ancestor, isLinkDisabled = false}: ThreadDividerProps) {
             ) : (
                 <PressableWithoutFeedback
                     onPress={() => {
-                        Timing.start(CONST.TIMING.SWITCH_REPORT);
```

Comment:
Same as before, isn't this the right place?

Reply:
`ThreadDivider` is performs `goBack` action and it won't measure the report switching time correctly (the timer will remain active till another report is open). Here is a reply with more details https://github.com/Expensify/App/pull/46966#issuecomment-2273539824

Reply:
oh clear, my bad! Thank you :)